src/Core/Framework/Plugin/PluginLifecycleService.php

```diff
@@ -508,4 +528,25 @@ private function getPluginInstance(string $pluginBaseClassString): Plugin
 
         return $this->getPluginBaseClass($pluginBaseClassString);
     }
+
+    private function savePluginConfiguration(Plugin $plugin, InstallContext $context): void
```

Comment:
we think, that this method should belong to the [SystemConfigService](https://github.com/shopware/platform/blob/master/src/Core/System/SystemConfig/SystemConfigService.php)
which should then be called here in the PluginLifecycleService

Reply:
I have fixed this.